TodoWithControllersAuthJWT/AuthController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace TodoWithControllersAuthJWT
+{
+    [ApiController]
+    [Route("/api/auth")]
+    [AllowAnonymous]
+    public class AuthController: ControllerBase
+    {
+        private readonly IAuthService _authService;
+
+        public AuthController(IAuthService authService)
+        {
+            _authService = authService ?? throw new ArgumentNullException(nameof(authService));;
+        }
+        [HttpGet("token")]
+        public IActionResult GenerateToken([FromQuery] string username, [FromQuery] string password)
```

Comment:
This is basically a ghetto version of basic auth. We should use @blowdart's basic auth handler 😄